gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Mon Mar 20 18:09:37 CET 2017
+#Sat Apr 29 11:24:25 CEST 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.4.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.5-all.zip
```

Comment:
Thanks for the reference.  If I remember, I'll dig through history to see if I can find a commit where it switched from `bin` to `all`.  Otherwise, I guess "it's just always been that way." :smiley:

> Eclipse changes this back to bin every time...

The Eclipse Gradle Buildship plugin is probably just delegating to the Gradle `wrapper` task, which defaults to `bin`.  If you do the upgrade from the command line, you can get it to use `all` directly by specifying `--distribution-type=ALL`.  For example:

```
$ ./gradlew wrapper --distribution-type=ALL --gradle-version=3.5
```

See `./gradlew help --task wrapper` for more information.